modules/xmpp/moderator.js

```diff
@@ -210,13 +212,33 @@ Moderator.prototype.createConferenceIq = function() {
                 value: this.options.conference.minBitrate
             }).up();
     }
-    if (this.options.conference.openSctp !== undefined) {
-        elem.c(
-            'property', {
-                name: 'openSctp',
-                value: this.options.conference.openSctp
-            }).up();
+
+    let openSctp;
+
+    // TODO: I don't know if this.options.conference is the same object as
+    // options.config in JitsiConference.js.
+    switch (this.options.conference.openBridgeChannel) {
+    case 'datachannel':
+    case true:
+    case undefined:
+        openSctp = true;
+        break;
+    case 'websocket':
+        openSctp = false;
+        break;
+    }
+
+    // NOTE: Edge does not yet implement DataChannel.
+    if (RTCBrowserType.isEdge() && openSctp) {
```

Comment:
Can you please add a `RTCBrowserType.supportsDataChannels` instead? It will be easier to handle in case Edge adds support for them in a later version.